velox/functions/prestosql/MapTopN.h

```diff
@@ -60,11 +60,6 @@ struct MapTopNFunction {
       return;
     }
 
-    if (n >= inputMap.size()) {
-      out.copy_from(inputMap);
```

Comment:
The problem is we don't want to throw if the value itself is null, but only if it's a complex type that contains null.  There's a way we could do this by accessing the key and element Vectors and iterating over them checking if they are null and if not, checking if they contain null.  But assuming N is generally small the tradeoff between complexity and efficiency probably isn't worth it.